common/saber_base.h

```diff
@@ -38,10 +38,19 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(POWERSAVE)                    \
     DEFINE_EFFECT(BATTERY_LEVEL)                \
     DEFINE_EFFECT(FAST_ON)                      \
+    DEFINE_EFFECT(QUOTE)			\
+    /* Fett263 Prop Effects */                  \
```

Comment:
No, effects are not just for one prop.
Please remove this comment.

Reply:
ok, removed thought you'd ask me to add since they're all planned for my prop